### PR TITLE
Deploy stable tag on production only

### PR DESCRIPTION
**Pull request by rogeruiz**
screen shot 2018-09-20 at 3 51 05 pm

### CODE REVIEW

**Comment by rogeruiz** [comment]
@mogul mentioned this in Slack, https://gsa-tts.slack.com/archives/C0ENP71UG/p1537356924000100

**Comment by LinuxBozo** [comment]
If we're going to go ahead and track stable tag, no reason to not do that for staging as well. Let's simplify and leave the pipeline dependencies (`passed`) alone and just update existing resource, yeah? We can always test other branches and such using the customize docs in the README.

**Comment by rogeruiz** [comment]
A good idea. Yeah, I wasn't sure how I wanted to approach that. At first _I almost_ added a `- name: prebuild-npm-job`. 😁 🤦‍♂️

File: ci/pipeline.yml [hunk 1]
```diff
@@ -131,6 +131,7 @@ resources:
   type: git
   source:
     uri: https://github.com/cloudfoundry-incubator/stratos.git
+    tag_filter: stable
```
Comment (rogeruiz) [inline ci/pipeline.yml]: Three commits for a single line change :trollface:

**Pull request merged**